src/main/java/org/usfirst/frc4904/robot/commands/intake/AxleIntakeOn.java

```diff
@@ -0,0 +1,10 @@
+package org.usfirst.frc4904.robot.commands.intake;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+
+public class AxleIntakeOn extends MotorConstant {
+    public AxleIntakeOn() {
+        super("Axel Intake On", RobotMap.Component.intakeAxleMotor, 0.5); // todo: needs motor speed
```

Comment:
Generally the `name` argument should be in camel caps without spaces (see other).